qos_bandwidth.adoc

```diff
@@ -30,7 +30,7 @@ The reset value is `UNSPECIFIED` for all other register fields.
 
 The bandwidth controllers at reset must allocate all available bandwidth to
 `RCID` value of 0. When the bandwidth controller supports bandwidth allocation
-per access-type, the access-type value of 0 of `RCID=0` is allocated all
+per access-type, the access-type value of 0 of `RCID=0` is allocated to all
```

Comment:
@kersten1 It should be - RCID=0, at=0 combination is allocated all available bandwidth at reset. This change makes it seem like the AT=0/RCID=0 is what gets allocated - here AT=0/RCID=0 is the subject and bandwidth is the object.